src/components/sortableList/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,85 @@
+import _ from 'lodash';
+import React, {useCallback} from 'react';
+import Text from '../../text';
+import View from '../../view';
+import SortableList from '../index';
+import {ComponentDriver, SortableListItemDriver} from '../../../testkit';
+
+const defaultProps = {
+  testID: 'sortableList'
+};
+
+const ITEMS = _.times(5, index => {
+  return {
+    text: `${index}`,
+    id: `${index}`
+  };
+});
+
+const TestCase = props => {
+  const renderItem = useCallback(({item}) => {
+    return (
+      <Text center testID={`item${item.id}`}>
+        {item.text}
+      </Text>
+    );
+  }, []);
+
+  return (
+    <View flex>
+      <View flex useSafeArea>
+        <SortableList data={ITEMS} renderItem={renderItem} {...defaultProps} {...props}/>
+      </View>
+    </View>
+  );
+};
+
+const _TestCase = props => <TestCase {...props}/>;
```

Comment:
What do you need this const for? 
you can use the TestCase itself..

Reply:
I copied from Dialog's test, where it caused an error (I think something about hooks location); just tested and it looks like it's passing, so I removed from both.